hosted/json-schemas/offering.schema.json

```diff
@@ -1,82 +1,152 @@
-  "definitions": {
-    "CurrencyDetails": {
+  "type": "object",
+  "properties": {
+    "additionalProperties": false,
+    "description": {
+      "type": "string",
+      "description": "Brief description of what is being offered."
+    },
+    "payin": {
       "type": "object",
       "additionalProperties": false,
       "properties": {
         "currencyCode": {
           "type": "string",
           "description": "ISO 3166 currency code string"
         },
-        "minAmount": {
+        "min": {
           "$ref": "definitions.json#/definitions/decimalString",
           "description": "Minimum amount of currency that can be requested"
         },
-        "maxAmount": {
+        "max": {
           "$ref": "definitions.json#/definitions/decimalString",
           "description": "Maximum amount of currency that can be requested"
+        },
+        "methods": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "additionalProperties": false,
+            "properties": {
+              "kind": {
+                "type": "string",
+                "description": "The type of payment method. e.g. BITCOIN_ADDRESS, DEBIT_CARD, etc."
+              },
+              "name": {
+                "type": "string",
+                "description": "Payment Method name. Expected to be rendered on screen."
+              },
+              "description": {
+                "type": "string",
+                "description": "Blurb containing helpful information about the payment method. Expected to be rendered on screen. e.g. \"segwit addresses only\""
+              },
+              "group": {
+                "type": "string",
+                "description": "value that can be used to group specific payment methods together e.g. Mobile Money vs. Direct Bank Deposit"
+              },
+              "requiredPaymentDetails": {
+                "type": "object",
+                "description": "A JSON Schema containing the fields that need to be collected in order to use this payment method"
+              },
+              "min": {
+                "$ref": "definitions.json#/definitions/decimalString",
+                "description": "minimum amount required to use this payment method."
+              },
+              "max": {
+                "$ref": "definitions.json#/definitions/decimalString",
+                "description": "maximum amount allowed when using this payment method."
+              },
+              "fee": {
+                "$ref": "definitions.json#/definitions/decimalString",
+                "description": "Fee charged to use this payment method. absence of this field implies that there is no _additional_ fee associated to the respective payment method"
+              }
+            },
+            "required": ["kind"]
+          }
         }
       },
-      "required": ["currencyCode"]
+      "required": ["currencyCode", "methods"]
```

Comment:
did we decide to change `currencyCode` to `currency`?

Reply:
no,  we stayed with currencyCode.

![Screenshot_20240313_120117_Chrome](https://github.com/TBD54566975/tbdex/assets/15972783/e5533202-50d7-4b45-b64b-1afe47ffa792)

https://www.iban.com/currency-codes